.github/workflows/ci.yml

```diff
@@ -16,23 +16,6 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Set up test environment
-        run: |
-          sudo apt-get update
-          sudo apt-get install xvfb gettext
-          wget https://github.com/mozilla/geckodriver/releases/download/v0.25.0/geckodriver-v0.25.0-linux64.tar.gz
-          mkdir geckodriver
-          tar -xzf geckodriver-v0.25.0-linux64.tar.gz -C geckodriver
-          export PATH=$PATH:$PWD/geckodriver
-          export BOKCHOY_HEADLESS=true
-
-      - run: geckodriver --version
-
       - name: Install Requirements
         run: |
           pip install -r requirements/ci.txt
-
-      - name: Run Tests
```

Comment:
@farhan it did not affect, I found there is some issue due to which ci.yml is not executing even in some past PRs, I am creating a separate issue so that we can properly find the reason to fix it. 
https://github.com/orgs/openedx/projects/55/views/1?pane=issue&itemId=42628017